Frameworks/CoreBluetooth/CBCentral.mm

```diff
@@ -25,7 +25,7 @@ @implementation CBCentral
 */
 - (id)copyWithZone:(NSZone*)zone {
     UNIMPLEMENTED();
-    return [self retain];
+    return self;
```

Comment:
Yes, this is arc enabled and technically i can say interoperable, but the class as a whole is stub (no one is setting these properties), so it would be a shame to call this interoperable.

---
In reply to: [76349060](https://github.com/Microsoft/WinObjC/pull/906#discussion_r76349060) [](ancestors = 76349060)